src/gui/categoryfiltermodel.cpp

```diff
@@ -181,6 +181,12 @@ CategoryFilterModel::CategoryFilterModel(QObject *parent)
     connect(session, &Session::categoryRemoved, this, &CategoryFilterModel::categoryRemoved);
     connect(session, &Session::torrentCategoryChanged, this, &CategoryFilterModel::torrentCategoryChanged);
     connect(session, &Session::subcategoriesSupportChanged, this, &CategoryFilterModel::subcategoriesSupportChanged);
+    connect(session, &Session::finishedStartingUp, this,
+            [this, session]()
+    {
+        for (const auto &handle : session->torrents().values())
```

Comment:
add `qAsConst()`?
or use iterator to loop, this avoids redundant traverse over the container, although I like the use of values() :P

Reply:
Sure. I am still confused about those macros. Eventually I'll get the hang of them.